rover/Profile.proto

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+package skywalking.v3;
+
+option java_multiple_files = true;
+option java_package = "org.apache.skywalking.apm.network.rover.profile.v3";
+option go_package = "skywalking.apache.org/repo/goapi/collect/rover/profile/v3";
+
+import "common/Common.proto";
+
+// Define the Rover Process profile task and upload profile data.
+service ProfileService {
+    // Query profile (start or stop) tasks
+    rpc queryTasks (ProfileTaskQuery) returns (Commands) {
+    }
+
+    // collect profile data
+    rpc collectProfileData (stream ProfileData) returns (Commands) {
```

Comment:
Profile existed for a while. Let's change all these to `EBPFProfilingService` with all relative things.